src/pointer-analysis/value_set.h

```diff
@@ -82,7 +82,12 @@ class value_sett
     size_t size() const { return data.size(); }
     bool empty() const { return data.empty(); }
 
+    void erase(unsigned i) { data.erase(i); }
```

Comment:
Now that we have a few methods that could use it, consider adding a typedef for `typename data_typet::key_type` and using it where appropriate. Not too important though.